languages/rust/reference/merge-strategy.md

```diff
@@ -0,0 +1,33 @@
+# merging strategy
+
+The Rust track maintainers [have decided](https://github.com/exercism/v3/discussions/1725#discussion-7438) to write down our merging strategy to communicate expectations and encourage quality contributions.
+
+## general principles
+
+- we strive to accept _all_ contributions even when **not** perfect.
+  - We adopt this principle of optimistic merging / Google engineering's code review handbook. See our [original discussion](https://github.com/exercism/v3/discussions/1725#discussion-7438) for appropriate sources.
```

Comment:
I note that these first two lines do not adopt principle 2 from #1725, but principle 3. I still think that's a bad idea.